apps/os/backend/auth/integrations.ts

```diff
@@ -73,6 +75,158 @@ export const integrationsPlugin = () =>
   ({
     id: "integrations",
     endpoints: {
+      // MCP OAuth callback endpoint
```

Comment:
This is a little above my paygrade, so this is just to say I haven't carefully reviewed this part